config/initializers/dfe_analytics.rb

```diff
@@ -0,0 +1,60 @@
+DfE::Analytics.configure do |config|
+  # Whether to log events instead of sending them to BigQuery.
+  #
+  # config.log_only = true
+
+  # Whether to use ActiveJob or dispatch events immediately.
+  #
+  # config.async = true
+
+  # Which ActiveJob queue to put events on
+  #
+  # config.queue = :default
```

Comment:
Shouldn't be better to have a specific queue for these events?
ie: `config.queue = :dfe_analytics`